src/renderer/lib/state.js

```diff
@@ -70,15 +70,12 @@ function load (cb) {
     if (err) {
       console.log('Missing config file: Creating new one')
     }
-    onSavedState(null, err ? {} : saved)
+    onSavedState(err ? {} : saved)
   })
 
-  function onSavedState (err, saved) {
```

Comment:
`err` is always null, so no need for this parameter or doing `cb(err)` since it's dead code.